### ISSUE TITLE
Same images are rendered in slightly different sizes

### ISSUE THREAD

**Issue by tapulator** [dompdf/dompdf#1126]
Hi, folks!

My HTML document contains images lined-up in one row without any space between them. In the generated PDF these images overlap slightly, so details like a fine border may disappear.

I pumped up the resolution of the images, thus increasing the relative DPI as recommended in [this comment](https://github.com/dompdf/dompdf/issues/685#issuecomment-20816718), but some images still have a slightly other size than specified.

Is there a way to avoid this image blur?

PS: Maybe you want to check out the [files](https://github.com/dompdf/dompdf/files/145804/test.zip) I used to reproduce and verify the problem. In test.pdf you see that the right green image is 1px wider than the left one.

**Comment by bsweeney** [comment]
There does appear to be a slight placement issue with the images. I can't quite find a way to work around the issue. I'm guessing the issue is more about the calculation used to determine placement of the images is not accurate enough to maintain such a tight arrangement.

**Comment by simonberger** [comment]
I was searching for an issue with the problem I just encountered and I think this is the same reason.
For testing purpose I used a div with a 1px border around.
`AbstractRenderer::_background_image` is called with width/height values including this border. Especially with activated repeat the result can differ much from the browser view.
I don't know where this issue should be solved and which other attributes could have a similar impact. Padding and Marging are working correctly.
